### ISSUE TITLE
kubeconfig.loadFromCluster() not working in 8.3.5

### ISSUE THREAD

**Issue by juanjoDiaz** [godaddy/kubernetes-client#560]
In version 8.3.5
```js
const kubeconfig = new KubeConfig();
kubeconfig.loadFromCluster();
```
produce the following error
```sh
Error: Failed to get /openapi/v2 and /swagger.json: forbidden: User "system:anonymous" cannot get path "/swagger.json"
```

it works fine in 8.3.4.

**Comment by bashofmann** [comment]
I can verify that it works with 8.3.4 and it is broken with 8.3.5.

**Comment by silasbw** [comment]
I'm having trouble reproing.  What is your cluster setup like?

**Comment by juanjoDiaz** [comment]
I can repro with a single node Kubernetes 1.15 cluster created with Kubeadm and mostly default settings.

**Comment by silasbw** [comment]
yeah, i just repo'd in a related project.  i'll dig in.

**Comment by Stono** [comment]
Hey
Any change we could get an 8.3.6 posted with a revert of this?
People across my org keep updating this module and getting caught out by it!

**Comment by silasbw** [comment]
yes, working on it now.

**Comment by silasbw** [comment]
Should be fixed with `8.3.6`.  Can you confirm?

**Comment by abz53378** [comment]
Thanks, `8.3.6` works well, 👍
